Atomic/syscontainers.py

```diff
@@ -2415,9 +2415,6 @@ def container_exec(self, name, detach, args):
 
         tty = os.isatty(0)
 
-        if util.is_user_mode():
-            raise ValueError("Command not supported in user mode")
```

Comment:
Hi, I am slightly confused with the `_get_oci_runtime` function. This question is a bit similar to the previous question. Could the run time that we get from `_get_oci_runtime` still be bwrap? If so, the command will fail later because bwrap does not handle exec? (guessing).

Reply:
yes, but more in general we allow to override it with anything else.  The exec could still fail or not be implemented as in the `bwrap-oci` case. 

In any case, since we don't know if the runtime supports exec or not, we should not check for it here.